www/install/php/Update-22.10.0-beta.1.php

```diff
@@ -35,8 +35,12 @@
 
     $errorMessage = "Unable to delete 'appKey' information from database";
     $pearDB->query("DELETE FROM `informations` WHERE `key` = 'appKey'");
-
     $pearDB->commit();
+
```

Comment:
Not related to this file but :
 https://github.com/centreon/centreon/blob/develop/src/CentreonRemote/Application/Webservice/CentreonConfigurationRemote.php#L567 
 
 This line should also be removed as actually, it is impossible to add a remote server due to a SQL error because the column doesn't exist anymore